.install/getrust.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Download and install rustup
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+# Source the cargo environment script to update the PATH
+source "$HOME/.cargo/env"
+
+# Install and set the required Rust version
+rustup install 1.72.0
```

Comment:
I'd just stick to the latest available Rust version, not 1.72. Ideally, we want to check more: 1. the latest compatible Rust edition 2. The latest available Rust (stable) 3. The latest available Rust (nightly)